From db33fbf55700507d6e67078f3d6b7ac610123429 Mon Sep 17 00:00:00 2001
From: Brahadambal Srinivasan <latha@linux.vnet.ibm.com>
Date: Wed, 1 Apr 2020 15:01:43 +0530
Subject: [PATCH] lsvpd: Add version and debug options to man page

The lsvpd man page and --help are not in sync. The version and
debug options were missing. This patch adds the two options to the
man page.

Signed-off-by: Brahadambal Srinivasan <latha@linux.vnet.ibm.com>
Reported-by: Shirisha Ganta <shiganta@in.ibm.com>
Reviewed-by : Shivaprasad G Bhat <sbhat@linux.ibm.com>
[Updated -v section - Vasant]
Signed-off-by: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
---
 man/lsvpd.8 | 22 ++++++++++++++++++++--
 1 file changed, 20 insertions(+), 2 deletions(-)

diff --git a/man/lsvpd.8 b/man/lsvpd.8
index 7030815..de5f3c3 100644
--- a/man/lsvpd.8
+++ b/man/lsvpd.8
@@ -24,7 +24,17 @@ lsvpd \- list Vital Product Data (VPD)
 .ad l
 .hy 0
 .HP 6
-\fBlsvpd\fR  [\fB\-l<device>\fR | \fB\-\-list=<device>\fR] [\fB\-m\fR | \fB\-\-mark\fR] [\fB\-s<serial#>\fR | \fB\-\-serial=<serial#>\fR] [\fB\-t<type/model#>\fR | \fB\-\-type=<type/model#>\fR] [\fB\-p<database\-directory>\fR | \fB\-\-path=<database\-directory>\fR] [\fB\-z<database\-archive>\fR | \fB\-\-zip=<database\-archive>\fR] [\fB\-h\fR | \fB\-\-help\fR]
+\fBlsvpd\fR  [\fB\-l<device>\fR | \fB\-\-list=<device>\fR]
+ [\fB\-m\fR | \fB\-\-mark\fR]
+ [\fB\-s<serial#>\fR | \fB\-\-serial=<serial#>\fR]
+ [\fB\-t<type/model#>\fR | \fB\-\-type=<type/model#>\fR]
+ [\fB\-p<database\-directory>\fR | \fB\-\-path=<database\-directory>\fR]
+ [\fB\-z<database\-archive>\fR | \fB\-\-zip=<database\-archive>\fR]
+ [\fB\-D\fR | \fB\-\-debug\fR]
+.HP 6
+\fBlsvpd\fR  [\fB\-v\fR | \fB\-\-version\fR]
+.HP 6
+\fBlsvpd\fR  [\fB\-h\fR | \fB\-\-help\fR]
 .ad
 .hy
 
@@ -66,6 +76,14 @@ Use <database\-archive> as the hardware inventory database, rather than the defa
 
 This option is useful for producing output for old hardware inventory databases that have been archived by the system\&.
 
+.TP
+\-D|\-\-debug
+Use this option to print extra information about devices (sysfs locations, etc)\&.
+
+.TP
+\-v|\-\-version
+Print the version of the command and exit.
+
 .TP
 \-h|\-\-help
 Display a help screen and quit\&.
@@ -87,5 +105,5 @@ lscfg(8), lsmcode(8), lsvio(8), vpdupdate(8)
 Eric Munson <ebmunson@us\&.ibm\&.com>\&. Brad Peters <bpeters@us\&.ibm\&.com>\&.
 
 .PP
-Copyright (c) 2002 - 2013 IBM\&.
+Copyright (c) 2002 - 2020 IBM\&.
